test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServer.java

```diff
@@ -88,4 +107,49 @@ public void shutdown() throws Exception {
 
         FileUtils.deleteDirectory(workingDirectory);
     }
+
```

Comment:
Any chance you can update the PR to only include the changes that are necessary?